weave/panels_py/panel_example_custom_board.py

```diff
@@ -0,0 +1,78 @@
+import weave
+from .. import weave_internal as internal
+from .. import weave_types as types
+from .generator_templates import template_registry
+
+
+# IMPORTANT: To enable this (or any template), it must be imported in
+# weave/panels_py/__init__.py This example is explicitly commented out since it
```

Comment:
Longer term it would be good to have less disabled/unfinished code in the repo. Ran into this with the routing and recent boards stuff - the tradeoffs between fixing stuff that isn't live vs. letting it rot are not always clear.

Reply:
In this case it is intended to be an example for MLEs to copy paste. Maybe this should just be a md file?